packages/google_maps_flutter/google_maps_flutter/android/src/main/java/io/flutter/plugins/googlemaps/GoogleMapController.java

```diff
@@ -42,10 +42,7 @@
 import com.google.android.gms.maps.model.Polygon;
 import com.google.android.gms.maps.model.Polyline;
 import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding;
-import io.flutter.plugin.common.BinaryMessenger;
-import io.flutter.plugin.common.MethodCall;
-import io.flutter.plugin.common.MethodChannel;
-import io.flutter.plugin.common.PluginRegistry;
+import io.flutter.plugin.common.*;
```

Comment:
Ditto

Reply:
i have this warning if i just import PluginRegistry : 

```
warning: [deprecation] PluginRegistry in io.flutter.plugin.common has been deprecated
import io.flutter.plugin.common.PluginRegistry;
```

Reply:
@xster @amirh @blasten - do any of you know if the maps plugin still needs deprecated functionality from PluginRegistry? I have some vague memory of it being special for some reason.

Reply:
@dnfield i fixed using  `io.flutter.plugin.common.PluginRegistry.Registrar`

Reply:
Correct, you can't import deprecated classes. You have to use it by the fully qualified name and add the lint ignore statement on the same line.